From 5ef345d6d36e09170006eb8761ef3811b553f74e Mon Sep 17 00:00:00 2001
From: Nathanael Noblet <nathanaelnoblet@gmail.com>
Date: Thu, 5 Jan 2012 18:25:59 -0500
Subject: [PATCH 1/2] lib: added unistd.h includes for gcc 4.7

Fedora 17 will include gcc-4.7 and as such the packages have all been
rebuilt. Barry failed because of a change in gcc. The following patch fixes
the issue.
---
 src/j_jdwp.cc |    2 +-
 src/packet.h  |    1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/j_jdwp.cc b/src/j_jdwp.cc
index 875f226..279e61c 100644
--- a/src/j_jdwp.cc
+++ b/src/j_jdwp.cc
@@ -24,7 +24,7 @@
 #include <sstream>
 #include <iomanip>
 #include <errno.h>
-
+#include <unistd.h>
 
 namespace Barry { namespace JDWP {
 
diff --git a/src/packet.h b/src/packet.h
index a89cf1d..52f16f2 100644
--- a/src/packet.h
+++ b/src/packet.h
@@ -26,6 +26,7 @@
 
 #include <string>
 #include <stdint.h>
+#include <unistd.h>
 #include "protocol.h"
 #include "data.h"
 
-- 
1.7.10.4

